docs/using.rst

```diff
@@ -44,17 +24,62 @@ above method instead.
 
 .. _prerequisites:
 
-Prerequisites
-=============
+Installation and Usage
+======================
+
+To install and run the client you just need to type:
+
+.. code-block:: shell
+
+   ./letsencrypt-auto
+
+.. warning:: Please do **not** use ``python setup.py install``.  That mode of
+             operation might corrupt your operating system and is **not supported**
+             by the Let's Encrypt team!
```

Comment:
I see, but in my long IRC support experience, I've seen a lot of them jumping straight to `pip install .` (without reading much docs), which is becoming more standard in Python world.

Also, you removed the part about forbidden use of `sudo` or `root`. While it's still true for all previously mentioned cases, this also applies to the wrapper: `sudo ./letsencrypt-auto` should not be called.